superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -458,6 +460,28 @@ const FiltersConfigForm = (
     forceUpdate();
   };
 
+  const validatePreFilter = () =>
+    setTimeout(
```

Comment:
curious: why is the setTimeout needed?

Reply:
Ahhh, I see the `// execute after render` comment - that's fine I guess, but would a `useEffect` hook be an option here?

Reply:
I can investigate if this is possible in the next iteration.